.github/workflows/on-push-to-master.yaml

```diff
@@ -22,9 +22,8 @@ jobs:
     
     - name: Install package
       run: |
-        (npm ci && npm audit --audit-level high && npm test) & \
-        (cd packages/ui && npm ci && npm audit --audit-level high && npm test) & \
-        (cd packages/api && npm ci && npm audit --audit-level high && npm test)
+        (npm ci && npm audit --audit-level critical && npm test) & \
+        (cd packages/ui && npm ci && npm audit --audit-level critical && npm test)
```

Comment:
Ideally, we'd leave this as "high", but in reality, it causes way too many blockers when waiting on dependencies to update their downstream dependencies.